### DISCUSSION TITLE
WebApp or something for QR code gen on a Linux Mac with no Web access

### DISCUSSION THREAD

**Discussion by NohWayJose** [gopro/labs#364] [Feature Requests]
I'm going diving for a few days well off the coast and am taking my laptop. It's an old Mac but I've put Linux on it to prolong its life (no point throwing it away just because Apple won't support older hardware). I want to be able to generate and print QR codes, as I refine my underwater GoPro skills. Is there a build of your QR software for Linux or as a WebApp or something, that I can put on the LinuxMac, that'll generate QR codes but which isn't reliant on the Interweb?

**Comment by dnewman-gpsw** [comment]
All these page can be run locally. https://gopro.github.io/labs/control/custom/  The QR code generation is just javascript.
